### PR TITLE
Optimize usage forms queries

### PR DESCRIPTION
**Pull request by emord**

### CODE REVIEW

File: custom/icds_reports/utils/aggregation_helpers/monolith/agg_awc.py [hunk 1]
```diff
@@ -363,14 +364,16 @@ def updates(self):
                 sum(due_list_ccs) AS usage_num_due_list_ccs,
                 sum(due_list_child) AS usage_num_due_list_child_health
             FROM "{usage_table}"
-            WHERE month = %(start_date)s GROUP BY awc_id, month
+            WHERE month = %(start_date)s AND form_date >= %(start_date)s AND form_date < %(next_month)
```
Comment (emord) [inline custom/icds_reports/utils/aggregation_helpers/monolith/agg_awc.py]: I intentionally did not change this, for the distributed query as month has an index on it and we don't have partitions so `form_date` filtering would have no effect

**Review by calellowitz — APPROVED** [review]
you are going to have to run `./manage.py update_icds_aggregation_output` but looks good

**Comment by emord** [comment]
real failure:

```
======================================================================
FAIL: custom.icds_reports.tests.test_aggregation_script:InactiveAWWsTest.test_aggregate_query
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/mnt/commcare-hq-ro/custom/icds_reports/tests/test_aggregation_script.py", line 489, in test_aggregate_query
    self.assertEquals(records.count(), 46)
AssertionError: 0 != 46
----------------------------------------------------------------------
```

**Comment by sravfeyn** [comment]
I think the only relevant piece that affects this timeframe is the pillow lag, if we have a lag of say 2 weeks for whatever reason, we would want to have this number high enough to process the 2 weeks data. I made it one month for now https://github.com/dimagi/commcare-hq/pull/25550/commits/9e9e1ac192936f47f106884243cce23abec2c697

**Pull request merged**